onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -8293,7 +8291,8 @@ def aten_swapdims(self: TensorType, dim0: int, dim1: int) -> TensorType:
 @torch_op("aten::sym_size.int", trace_only=True)
 def aten_sym_size(self: TensorType, dim: int = 0) -> INT64:
     """sym_size.int(Tensor self, int dim) -> SymInt"""
-    return op.Squeeze(op.Shape(self, end=dim + 1, start=dim))
+
+    return op.Gather(op.Shape(self), dim, axis=0)
```

Comment:
If dim is int, we should use op.Shape(x, start=, end=...)

Reply:
Done